stdlib/asm/crypto/stark/deep_queries.masm

```diff
@@ -293,12 +292,12 @@ proc.compute_q_z_p_gz_at_alpha
     ### a) Load alpha^77
     swapdw
     drop drop
-    exec.constants::deep_rand_alpha_77
+    exec.constants::deep_rand_alpha_77_ptr
     mem_loadw
 
     ### b) Load alpha^2
     push.0.0
-    exec.constants::deep_rand_alpha_2
+    exec.constants::deep_rand_alpha_2_ptr
     mem_loadw
```

Comment:
Similar to the last comment (and maybe for a future PR), but this could be:
```
exec.constants::get_deep_rand_alpha_2
```

Reply:
Makes sense, we will for sure have subsequent PRs with more cleanups/refactorings